### ISSUE TITLE
Colours are not correct

### ISSUE THREAD

**Issue by edwardsdean** [vector-im/element-web#25901]
### Steps to reproduce

1. Where are you starting? What can you see?
Element version: 1.11.37
Olm version: 3.2.14
Light version

Text colour and display name colours have changed in the newest update, the text is no longer solid black colour and display name colours are very odd looking low saturation?

Having the text no longer a solid black makes it difficult for me to read.

3. What do you click?

5. More steps…
From the app with expected colours
![Element App](https://github.com/vector-im/element-desktop/assets/8325836/d94d4092-7e59-49a1-bffa-1bb59e472b21)

From the desktop app with the weird colours
![Element Desktop](https://github.com/vector-im/element-desktop/assets/8325836/c8e36bd3-8140-438e-a6fc-e67d822b8f93)

### Operating system

Windows

### Application version

Element version: 1.11.37 Olm version: 3.2.14

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by KarimJedda** [comment]
@edwardsdean is right. 

In your last change you might have tested it only against dark mode.

The following is not accessible:

![Screenshot from 2023-08-02 10-27-46](https://github.com/vector-im/element-web/assets/5029053/68ef8be4-c164-497e-bbed-c2c51efca75d)


![Screenshot from 2023-08-02 10-30-44](https://github.com/vector-im/element-web/assets/5029053/ed5251e9-899e-40b8-8974-1e767c02cc27)

**Comment by germain-gg** [comment]
What is mentioned in this comment  https://github.com/vector-im/element-web/issues/25901#issuecomment-1661751607 is totally unrelated to the topic of this issue. Please raise that in a different ticket.

As @t3chguy mentioned. The colour update is intentional and not a regression
